.pre-commit-config.yaml

```diff
@@ -5,6 +5,7 @@ repos:
     rev: v4.6.0
     hooks:
       - id: check-added-large-files
+        args: ["--maxkb=1500"]
```

Comment:
Jw, why do this? For the cassettes? You can alternately use `exclude`

Reply:
This is for the journal quality data -- I can exclude it specifically too.

Reply:
done!